style.css

```diff
@@ -171,88 +177,124 @@ video {
 
 .contact {
 	color: white;
-	background-color:#270483;
+	background-color: #270483;
 	padding-bottom: 60px;
 }
 
-.contact p {
-	font-family: 'Source Sans Pro', sans-serif;
-}
 footer {
 	color: white;
 	background-color: #270483;
-	font-family: 'Source Sans Pro', sans-serif;
 }
+
 footer p {
-	margin:0;
+	margin: 0;
+	font-size: 1rem;
 	
 }
 
 .projectBlocks {
+	display: -webkit-box;
+	display: -ms-flexbox;
 	display: flex;
+	-webkit-box-pack: space-evenly;
+	-ms-flex-pack: space-evenly;
 	justify-content: space-evenly;
+	-ms-flex-wrap: wrap;
 	flex-wrap: wrap;
 }
 
 .block {
```

Comment:
I missed this somehow the first time - should also change these class names to something like `project` and `projectsContainer`